npm/scripts/build.js

```diff
@@ -0,0 +1,46 @@
+const path = require("path");
+const {exec} = require("child_process");
+
+const cwd = process.cwd();
+
+const indexPath = path.relative(cwd, path.resolve(__dirname, "../index.js"));
+
+const jsPaths = [
+  "../../javascript",
+  "../../third_party/closure-library",
+  "../../third_party/grpc/third_party/protobuf/js",
+].map(jsPath => path.relative(cwd, path.resolve(__dirname, jsPath)));
+
+const grpcWebExports = [
+  "grpc.web.AbstractClientBase",
+  "grpc.web.ClientReadableStream",
+  "grpc.web.Error",
+  "grpc.web.GrpcWebClientReadableStream",
+  "grpc.web.GrpcWebStreamParser",
+  "grpc.web.Status",
+  "grpc.web.StatusCode",
+  "grpc.web.StreamBodyClientReadableStream",
+  "grpc.web.GrpcWebClientBase",
+
+  // @TODO: Include these
```

Comment:
Doesn't really need to. These 2 files are irrelevant for this purpose.

Reply:
Wouldn't `grpc.web.GatewayClientBase` be required if a client was compiled with `mode=base64`?

Reply:
We may have to remove it for some other reasons in the near future. So I just don't want to call this out as a `TODO`. We can add support for that any time.